src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -702,6 +717,54 @@ export default {
 			this.$emit('update:isReactionsMenuOpen', true)
 		},
 
+		async forwardToNote() {
+			const NoteToSelf = this.$store.getters.conversationsList.filter(conversation => conversation.type === CONVERSATION.TYPE.NOTE_TO_SELF)[0]
```

Comment:
Might by empty, so `[0]` will fail. In that case the function from the service file should be called